pkg/commands/compute/init.go

```diff
@@ -219,6 +221,26 @@ func (c *InitCommand) Exec(in io.Reader, out io.Writer) (err error) {
 		c.CloneFrom = from
 	}
 
+	defer func() {
```

Comment:
Looks like this only tracks users who pass the `--from` flag?

Is there a reason not to define this `defer` earlier in the `Exec()` function and to have it beacon for different types of users?

Reply:
We actually can't send beacons if the service ID doesn't exist yet, so if someone is running `init` without `--from=<service_id>` we don't have anywhere to even send the notification. It would certainly be possible to move this up—and simplify this condition—since `c.CloneFrom` is a service ID iff `isExistingService` is true.

Reply:
Gotcha. Makes sense.